CHANGELOG.md

```diff
@@ -6,14 +6,13 @@
 
 - Added `resource_attributes` config option to the `splunk_otel.start_tracing()` function.
   [#57](https://github.com/signalfx/splunk-otel-python/pull/57)
+- Added support for `OTEL_SERVICE_NAME`.
+  [#64](https://github.com/signalfx/splunk-otel-python/pull/64)
 
 ### Removed 
 
-- Removed `service_name` config option from the `splunk_otel.start_tracing()` function.
-  Please pass `resource_attributes={'service.name': 'my-service-name'}` to the function instead.
-  [#57](https://github.com/signalfx/splunk-otel-python/pull/57)
-- Removed support for `SPLUNK_SERVICE_NAME` environment variable.
-  Please use `OTEL_RESOURCE_ATTRIBTES=service.name=<my-service-name>` instead.
+- Deprecated support for `SPLUNK_SERVICE_NAME` environment variable.
+  Please use `OTEL_SERVICE_NAME=<my-service-name>` instead.
```

Comment:
This can be moved to `### Deprecated` section.
Reference: https://keepachangelog.com/en/1.1.0/

Reply:
Done. Thanks.